website/projects/tests/test_awssync_structs.py

```diff
@@ -0,0 +1,217 @@
+        ]
+
+        self.treelist = [
+            awssync.SyncData("email1@example.com", "project1", "Fall 2020"),
+            awssync.SyncData("email2@example.com", "project2", "Fall 2020"),
+            awssync.SyncData("email3@example.com", "project3", "Spring 2021"),
+            awssync.SyncData("email4@example.com", "project4", "Spring 2021"),
+        ]
+
+        self.aws_tree1 = awssync.AWSTree(
+            "AWS Tree",
+            "12345",
+            [
+                awssync.Iteration(
+                    "Fall 2020",
+                    "54321",
+                    [
+                        awssync.SyncData("email1@example.com", "project1", "Fall 2020"),
+                        awssync.SyncData("email2@example.com", "project2", "Fall 2020"),
+                    ],
+                ),
+                awssync.Iteration(
+                    "Spring 2021",
+                    "98765",
+                    [
+                        awssync.SyncData("email3@example.com", "project3", "Spring 2021"),
+                        awssync.SyncData("email4@example.com", "project4", "Spring 2021"),
+                    ],
+                ),
+            ],
+        )
+
+        self.aws_tree2 = awssync.AWSTree(
+            "AWS Tree",
+            "12345",
+            [
+                awssync.Iteration(
+                    "Fall 2020",
+                    "54321",
+                    [
+                        awssync.SyncData("email1@example.com", "project1", "Fall 2020"),
+                        awssync.SyncData("email2@example.com", "project2", "Fall 2020"),
+                    ],
+                ),
+                awssync.Iteration(
+                    "Spring 2021",
+                    "98765",
+                    [
+                        awssync.SyncData("email3@example.com", "project3", "Fall 2021"),
+                        awssync.SyncData("email4@example.com", "project4", "Spring 2021"),
+                    ],
+                ),
+            ],
+        )
+
+        self.aws_tree3 = awssync.AWSTree(
+            "AWS Tree",
+            "12345",
+            [
+                awssync.Iteration(
+                    "Fall 2020",
+                    "54321",
+                    [
+                        awssync.SyncData("email1@example.com", "project1", "Fall 2020"),
+                        awssync.SyncData("email2@example.com", "project2", "Fall 2020"),
+                    ],
+                ),
+                awssync.Iteration(
+                    "Fall 2020",
+                    "98765",
+                    [
+                        awssync.SyncData("email3@example.com", "project3", "Fall 2021"),
+                        awssync.SyncData("email4@example.com", "project4", "Spring 2021"),
+                    ],
+                ),
+            ],
+        )
+
+    def test_repr_AWSTree(self):
+        self.assertEquals(str(self.awstree), "AWSTree('Name', '1234', [])")
```

Comment:
why did i use the str function instead of just calling repr on the object. repr and str are closely related, but not always the same (repr is for debugging, so gives more precise output, str is for user interaction, and should look nicer (but is less precise))